subgraph/abis/PublicLock.json

```diff
@@ -1814,208 +1814,58 @@
     "sig": "function initialize()"
   },
   {
-    "inputs": [],
-    "name": "CANNOT_APPROVE_SELF",
-    "type": "error",
-    "sig": "error CANNOT_APPROVE_SELF()"
-  },
-  {
-    "inputs": [],
-    "name": "CANT_BE_SMALLER_THAN_SUPPLY",
-    "type": "error",
-    "sig": "error CANT_BE_SMALLER_THAN_SUPPLY()"
-  },
-  {
-    "inputs": [],
-    "name": "CANT_EXTEND_NON_EXPIRING_KEY",
-    "type": "error",
-    "sig": "error CANT_EXTEND_NON_EXPIRING_KEY()"
-  },
-  {
-    "inputs": [],
-    "name": "GAS_REFUND_FAILED",
-    "type": "error",
-    "sig": "error GAS_REFUND_FAILED()"
-  },
-  {
-    "inputs": [],
-    "name": "INSUFFICIENT_ERC20_VALUE",
-    "type": "error",
-    "sig": "error INSUFFICIENT_ERC20_VALUE()"
-  },
-  {
-    "inputs": [],
-    "name": "INSUFFICIENT_VALUE",
-    "type": "error",
-    "sig": "error INSUFFICIENT_VALUE()"
-  },
-  {
-    "inputs": [],
-    "name": "INVALID_ADDRESS",
-    "type": "error",
-    "sig": "error INVALID_ADDRESS()"
-  },
-  {
+    "constant": false,
```

Comment:
I think we can remove the change in ABI here, as this is a partial change for v12 that is not finalized here but was already checked in the master branch.

Reply:
ok so I'll remove this file!